project/make/.ensure-webserver

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+# Build a Go static web server on top of busybox image
+# and compile it for target daemon
+IMAGE="webserver"
+
+dir="$DEST/webserver"
+mkdir -p "$dir"
+(
+	cd "$dir"
+	cat > main.go <<'EOF'
+		package main
+
+		import (
+		  "log"
+		  "net/http"
+		)
+
+		func main() {
+		  fs := http.FileServer(http.Dir("/static"))
+		  http.Handle("/", fs)
+		  log.Panic(http.ListenAndServe(":80", nil))
+		}
```

Comment:
@cpuguy83 that'd require its own package and stuff like `go build ./...` would build that unnecessarily. I think this is just fine except I didn't get tianon's comment.